containers/azure-static-web-apps/.devcontainer/devcontainer.json

```diff
@@ -11,17 +11,20 @@
 	},
 	"forwardPorts": [ 7071, 4280 ],
 
-	// Set *default* container specific settings.json values on container create.
-	"settings": {},
-
-	// Add the IDs of extensions you want installed when the container is created.
-	"extensions": [
-		"ms-azuretools.vscode-azurefunctions",
-		"ms-azuretools.vscode-azurestaticwebapps",
-		"ms-dotnettools.csharp",
-		"ms-python.python",
-		"dbaeumer.vscode-eslint"
-	],
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+			"ms-azuretools.vscode-azurefunctions",
```

Comment:
Nit: Extension ids might be missing one indentation level.

Reply:
Good catch, fixed!